docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/StochasticGradientDescent.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.ML;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.BinaryClassification
+{
+    public static class StochasticGradientDescent
+    {
+        // In this examples we will use the adult income dataset. The goal is to predict
+        // if a person's income is above $50K or not, based on different pieces of information about that person.
```

Comment:
>based on different pieces  [](start = 54, length = 26)

"demographic information"

Reply:
that's a better wording. i changed to that.

---
In reply to: [259450893](https://github.com/dotnet/machinelearning/pull/2688#discussion_r259450893) [](ancestors = 259450893)